From: =?utf-8?q?Timo_R=C3=B6hling?= <roehling@debian.org>
Date: Tue, 7 May 2024 22:51:40 +0200
Subject: Use proper lapack dependency for __config__.py

---
 numpy/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/numpy/meson.build b/numpy/meson.build
index a8f810c..74ad3b2 100644
--- a/numpy/meson.build
+++ b/numpy/meson.build
@@ -345,7 +345,7 @@ conf_data.set('PYTHON_VERSION', py.language_version())
 # `np.show_config()`; needs some special handling for the case BLAS was found
 # but CBLAS not (and hence BLAS was also disabled)
 dependency_map = {
-  'LAPACK': lapack_dep,
+  'LAPACK': lapack,
 }
 if have_blas
   dependency_map += {'BLAS': blas}
